src/main/java/org/minions/devfund/lourdes/battleship/Ocean.java

```diff
@@ -0,0 +1,127 @@
+package org.minions.devfund.lourdes.battleship;
+
+import java.util.Random;
+
+public class Ocean {
+    private Ship[][] ship;
+    private int shotsFired;
+    private int hitCount;
+    private int shipsSunk;
+    private String[] shipType = {"Battleship", "BattleCruiser", "Cruiser", "LightCruiser", "Submarine"};
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "shipType" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)